ecmascript/transforms/base/src/hygiene/mod.rs

```diff
@@ -665,6 +665,21 @@ impl<'a> VisitMut for Hygiene<'a> {
         n.class.visit_mut_with(self);
     }
 
+    fn visit_mut_class_method(&mut self, n: &mut ClassMethod) {
+        let mut folder = Hygiene {
+            config: self.config.clone(),
+            current: Scope::new(ScopeKind::Fn, None),
+            ident_type: IdentType::Ref,
+            var_kind: None,
+        };
+
+        folder.ident_type = IdentType::Binding;
+        n.function.params.visit_mut_children_with(&mut folder);
+        folder.apply_ops(n);
+
+        n.visit_mut_children_with(self);
```

Comment:
Doesn't this visit `n.functions.params` again?
Is it intentional?

Reply:
I guessed similar, but somehow without this some of existing test fails like below

```
// tests\fixture\swc-node-335\case1\output\index.js

function _defineProperty(obj, key, value) {
    if (key in obj) {
        Object.defineProperty(obj, key, {
            value: value,
            enumerable: true,
            configurable: true,
            writable: true
        });
    } else {
        obj[key] = value;
    }
    return obj;
}
let Foo = class Foo {
    method() {
        class Foo {
        }
    }
};
_defineProperty(Foo, "a", 1);


Diff < left / right > :
 function _defineProperty(obj, key, value) {
     if (key in obj) {
         Object.defineProperty(obj, key, {
             value: value,
             enumerable: true,
             configurable: true,
             writable: true
         });
     } else {
         obj[key] = value;
     }
     return obj;
 }
 let Foo = class Foo {
     method() {
<        let Foo1 = class Foo {
<        };
>        class Foo {
>        }
     }
 };
 _defineProperty(Foo, "a", 1);
```

Trying to recalculate identifier, which makes output incorrect.

Reply:
I meant replacing this with manual visit of properties, not removing this line.
Did you tried it?

Reply:
Updated PR to manually request traverses like https://github.com/swc-project/swc/pull/1964/files#diff-10d7cd5c048de48351715805d282583190a251ec35e84f7c6a6e7f633387d6bdR681 and seems it makes test correctly passing, wonder if there's missing one or it's ok.

Reply:
`decorators` should be also visited as skipping it can break some Deno code.